src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -460,8 +459,8 @@ protected DynamoModel(StartConfiguration configuration)
             var libraryCore =
                 new ProtoCore.Core(new Options { RootCustomPropertyFilterPathName = string.Empty });
 
-            libraryCore.Executives.Add(Language.kAssociative, new Executive(libraryCore));
-            libraryCore.Executives.Add(Language.kImperative, new ProtoImperative.Executive(libraryCore));
+            libraryCore.Compilers.Add(ProtoCore.Language.kAssociative, new ProtoAssociative.Compiler(libraryCore));
```

Comment:
@ke-yu  Sure, ill place these in the enhancements task. This is so we can make the changes smaller
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193